<launch>
	<node name="mapper" type="mapper" pkg="ethzasl_icp_mapper" output="screen" >
		<remap from="cloud_in" to="/dynamic_point_cloud" />
		<!--remap from="cloud_in" to="/nifti_point_cloud" /-->
		<param name="subscribe_scan" value="false"/>
		<param name="icpConfig" value="$(find ethzasl_icp_mapper)/launch/nifti/icp.yaml" />
		<param name="inputFiltersConfig" value="$(find ethzasl_icp_mapper)/launch/nifti/input_filters.yaml" />
		<param name="mapPostFiltersConfig" value="$(find ethzasl_icp_mapper)/launch/nifti/map_post_filters.yaml" />
		<param name="odom_frame" value="/odom" />
		<param name="map_frame" value="/map" />
		<param name="useROSLogger" value="true" />
		<param name="minOverlap" value="0.15" /> 
		<param name="minMapPointCount" value="600000" /> <!-- quick fix to have map published each time -->
	</node>
</launch>
